webapp/channels/src/components/add_users_to_group_modal/add_users_to_group_modal.tsx

```diff
@@ -107,9 +107,9 @@ const AddUsersToGroupModal = (props: Props) => {
                     aria-label='Close'
                     onClick={goBack}
                 >
-                    <LocalizedIcon
+                    <i
                         className='icon icon-arrow-left'
-                        ariaLabel={{id: t('user_groups_modal.goBackLabel'), defaultMessage: 'Back'}}
+                        aria-label={formatMessage({id: 'user_groups_modal.goBackLabel', defaultMessage: 'Back'})}
```

Comment:
Lets remove the aria label from here and add it to its parent above, because the button will be the one in focus whos aria label will be read out instead of its icon inside of it.
```suggestion
                        
```


```ts
<button
                    type='button'
                    className='modal-header-back-button btn btn-icon'
                    aria-label={formatMessage({id: 'user_groups_modal.goBackLabel', defaultMessage: 'Back'})}
                    onClick={goBack}
                >
                ```

Reply:
Yeah, Correct. Thanks for the Help.
In the recent commit, the aria-label and es-lint issue is fixed.